concordia/static/js/action-app/index.js

```diff
@@ -136,8 +136,12 @@ export class ActionApp {
                     break;
                 }
                 case 'asset_reservation_obtained':
-                    // FIXME: we need to test whether the user who reserved it is different than the user we're running as!
-                    this.markAssetAsUnavailable(assetId);
+                    if (
+                        this.config.currentUser != 'None' &&
```

Comment:
I'd like this to be `if (this.config.currentUser && this.config.currentUser != message.user_pk)`

Reply:
Done